src/server-helpers.ts

```diff
@@ -19,18 +19,15 @@ type EquippedRequest = Request<
   { [metaSymbol]?: { logger: ActualLogger } }
 >;
 
-export type ChildLoggerExtractor = (request: Request) => ActualLogger;
+export type LoggerProvider = (request?: Request) => ActualLogger;
```

Comment:
naming should be less confusing with `ChildLoggerProvider`

Reply:
jsdoc also needed

Reply:
done in  [3a489d7](https://github.com/RobinTail/express-zod-api/pull/2167/commits/3a489d7cce2e2e279fba3ff2ce809869dac34e8d)